modules/sdk/src/main/scala/org/tessellation/sdk/http/routes/SnapshotRoutes.scala

```diff
@@ -91,6 +91,23 @@ final case class SnapshotRoutes[F[_]: Async: KryoSerializer, S <: Snapshot[_, _]
           }
         }.getOrElse(NotFound())
 
+    case GET -> Root / "hash" / SnapshotOrdinalVar(ordinal) :? FullSnapshotQueryParam(fullSnapshot) =>
+      import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+
+      if (!fullSnapshot) {
+        snapshotStorage.getHash(ordinal).flatMap {
+          Ok(_)
+        }
+      } else {
+        fullGlobalSnapshotStorage
+          .map(
+            _.readHash(ordinal)
+          )
+          .traverse(identity)
```

Comment:
`traverse(identity)` is just a `sequence` (and with additional `flatten` it might be a `flatSequence`. I'm not sure as I just deduct how it works by github code)

Reply:
Changed

```scala
          .map(
            _.readHash(ordinal)
          )
          .traverse(identity)
```
to

```scala
          .map(
            _.readHash(ordinal)
          )
          .flatSequence
```